sdk/nestjs/src/DevCycleModule/DevCycleService.ts

```diff
@@ -25,8 +25,8 @@ export class DevCycleService {
     variableValue<T extends DVCVariableValue>(
         key: string,
         defaultValue: T,
-    ): DVCVariableValue {
-        return this.devcycleClient.variableValue(
+    ): VariableTypeAlias<T> {
```

Comment:
could also just type this as the ReturnType of the client's variableValue method

Reply:
or not define it assuming that doesn't cause a linting error or something

Reply:
I'm fairly sure leaving it undefined will cause a lint error — updated to use the return type